polyfills/Array/prototype/find/config.toml

```
@@ -21,6 +21,7 @@ firefox = "<24"
 ie = "*"
 ie_mob = "10 - *"
 opera = "<32"
+op_mob = "<64"
```

```suggestion
op_mob = "<32"
```

as discussed here : https://github.com/Financial-Times/polyfill-library/pull/1078#issuecomment-892009972